src/filter_debris_by_perc.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import os
+import pickle
+from argparse import ArgumentParser
+
+import numpy as np
+import pandas as pd
+
+import utils
+
+def get_debris_perc(row, base_folder):
+    mask = np.load(os.path.join(base_folder, row.cropped_label))
+    img = np.load(os.path.join(base_folder, row.cropped_path))
+    img = np.moveaxis(img, -1, 0) # channels first
+    debris_mask = utils.get_debris_glaciers(img, mask)
+    return debris_mask.sum() / mask.sum()
+
+if __name__ == '__main__':
+    parser = ArgumentParser()
+    parser.add_argument("--toy_data", default=True, type=bool,
```

Comment:
why not make the parser argument just the path to the data? Then you can avoid the if else, and it's also more flexible.